applications/WindEngineeringApplication/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
I think sth has been updated in the CMakeLists template, can you check that it is up to date?

Reply:
Can you point me to that template? I haven't found what you mean.

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/templates/template_application/CMakeLists.txt.in

Reply:
updated; thanks